packages/hop-node/src/watchers/classes/Bridge.ts

```diff
@@ -580,6 +585,25 @@ export default class Bridge extends ContractBase {
         options.syncCacheKey
       )
       state = await this.db.syncState.getByKey(syncCacheKey)
+
+      const isHeadSync = syncCacheKey.includes(HeadSyncKeySuffix)
```

Comment:
suggestion: use `startsWith` instead of `includes`

```suggestion
const isHeadSync = syncCacheKey.startsWith(HeadSyncKeySuffix)
```

Reply:
Woah TIL about `startsWith`. In this case, the string is at the end so I'm going to change it to `endsWith`, but I like this a lot.

Reply:
Resolved with 2c22d36cd66155375efc707a3572fe8fda6fc1fc